From 8c2be447b7a17449f71c24b6b31f141abecbb514 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Mon, 23 Mar 2009 12:21:08 -0700
Subject: [PATCH] Staging: epl: remove some unused types

SHORT, USHORT, INT, UINT, LONG, and ULONG aren't used, so delete them.

Cc: Daniel Krueger <daniel.krueger@systec-electronic.com>
Cc: Ronald Sieber <Ronald.Sieber@systec-electronic.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/epl/global.h |   23 +++--------------------
 1 file changed, 3 insertions(+), 20 deletions(-)

--- a/drivers/staging/epl/global.h
+++ b/drivers/staging/epl/global.h
@@ -33,25 +33,7 @@
 
 #define TRACE  printk
 
-#ifndef SHORT
-#define SHORT short int
-#endif
-#ifndef USHORT
-#define USHORT unsigned short int
-#endif
-#ifndef INT
-#define INT int
-#endif
-#ifndef UINT
-#define UINT unsigned int
-#endif
-#ifndef LONG
-#define LONG long int
-#endif
-#ifndef ULONG
-#define ULONG unsigned long int
-#endif
-    // --- logic types ---
+// --- logic types ---
 #ifndef BYTE
 #define BYTE unsigned char
 #endif
@@ -64,7 +46,8 @@
 #ifndef BOOL
 #define BOOL unsigned char
 #endif
-    // --- alias types ---
+
+// --- alias types ---
 #ifndef TRUE
 #define TRUE  0xFF
 #endif
